api/clients/Admin/src/main.js

```diff
@@ -298,8 +300,11 @@ function sendCommand(command, selectedAntSet){
     });
 
     socket.emit('cmd', {
-        command: command,
-        to: antSims
+        token: PRIVATE.cmdToken,
```

Comment:
If I understand correctly, [`cmdToken` is sent to the dashboard client-side](https://github.com/anthill/pheromon/blob/d03e4ba89952ad0f1caa46ad13aabfcf874d9c4d/api/clients/Dashboard/src/main.js#L16). 
Is it intentional?
If not, what other unintentional things slipped?
We need a better management of tokens; the `PRIVATE.json` approach with implicit uses will bit us.

More importantly, anything that touches `PRIVATE.json` should be thoroughly documented in the PR, so anyone who reviews the PR can know what to focus on.
Who creates this token?
Who has it?
What is it used for?
What part of the application are affected/unaffected by this token?
What are the threats associated to having the token?

Even more importantly, we need an actual review process and block merging if a proper review hasn't happened. IMHO this PR shouldn't have been merged as is. More than one person should be responsible/accountable for merging something that sensitive.